docs/examples/water-value.ipynb

```diff
@@ -0,0 +1,541 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c087aa2b",
+   "metadata": {},
+   "source": [
+    "# Water Values for Long-Duration Storage Operation\n",
+    "\n",
+    "This example demonstrates how the concept of **water values** (known from hydroelectric power plants) can be used to capture seasonal behaviour of long-duration storage in a rolling horizon optimisation with limited operational foresight. Applying the concept of water values helps internalise the future value of stored energy beyond the optimisation horizon when making short-term operational decisions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8fdd50ae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "\n",
+    "import pypsa\n",
+    "\n",
+    "pypsa.options.params.optimize.solver_name = \"gurobi\"\n",
+    "logging.getLogger(\"gurobipy\").setLevel(logging.ERROR)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec3b7948",
+   "metadata": {},
+   "source": [
+    "For this example, we consider a simple energy system with a single bus and 3-hourly time steps over one year (8760 hours). We have time-varying load, wind and solar generation at zero marginal cost, a gas generator with marginal cost function $C'(g) = 80 + 0.01 g$ €/MWh, and hydrogen storage consisting of electrolysis, turbine, and a non-cyclic storage tank with a set initial energy. All components have fixed capacities; investments are not optimised here. "
```

Comment:
3-hourly means 2920 snapshots, so maybe say that in the brackets, instead of  `(8760)`, to avoid confusion